.eslintrc.js

```diff
@@ -6,5 +6,6 @@
  */
 
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
+  // extends: './node_modules/eslint-config-hackreactor/index.js'
```

Comment:
I think you could remove this line. I don't think we'll need to use the hackreactor styling guide for this project.

Reply:
good call, thanks.